.github/workflows/build.yml

```diff
@@ -32,3 +32,7 @@ jobs:
         run: sudo apt-get install -y genisoimage
       - name: Build with Maven
         run: mvn -B install --file pom.xml
+      - name: Build Docker image
+        uses: docker/build-push-action@v5
+        with:
+          tags: philippvk/minigolf:latest
```

Comment:
I guess I will have to add the access token as a repo secret?

Reply:
Is this pushing to docker.io or https://quay.io/repository/philippvk/minigolf?

Reply:
Only if you want CI to push the image to a Docker repo. In it's current form, CI will just build the image to make sure it doesn't break unexpectedly.